cmd/server.go

```diff
@@ -71,15 +71,15 @@ func RunServer(cmd *cobra.Command, args []string) { // coverage-ignore
 	rt := server.CreateRouter(serverCfg, log, validation.NewContentConfiguration())
 	rt.Handle("/metrics", metricsHandler)
 
-	server := &http.Server{
+	srv := &http.Server{
 		Addr:         ":" + serverCfg.ServerPort,
 		Handler:      rt,
 		ReadTimeout:  10 * time.Second,
 		WriteTimeout: 10 * time.Second,
 	}
 
 	go func() {
-		if err := server.ListenAndServe(); err != nil && err != http.ErrServerClosed {
+		if err := srv.ListenAndServe(); err != nil && err != http.ErrServerClosed {
```

Comment:
I would suggest to use `err != nil && !errors.Is(err, http.ErrServerClosed)` here